source/hierarchy.txt

```diff
@@ -269,20 +276,26 @@ characteristics, and growth expectations.
    applications, high-memory workloads, and high-CPU workloads, contact
    |mdb-support| for customized guidance.
 
-You can estimate 
-the {+cluster+} resources required by using your organization's approximate data size and workload:
+You can estimate the {+cluster+} resources required by using your organization's approximate data size and workload:
```

Comment:
instead of "required" I would say:Y ou can estimate the {+cluster+} resources that your deployment requires by